tests/integ/test_udf.py

```diff
@@ -545,6 +548,9 @@ def plus_one_month(x):
     session.clear_imports()
 
 
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC, reason="SNOW-609328: support caplog in SP regression test"
```

Comment:
caplog should work since pytest has been working, but somehow it's still failing. I guess it's related to python udf logging. So I created https://snowflakecomputing.atlassian.net/browse/SNOW-609328 for tracking.